deploy/e2e/gh-e2e-template/README.md

```diff
@@ -0,0 +1,37 @@
+# Usage
```

Comment:
can you remove if we don't have any comments?

Reply:
Eh.....I'm not sure I can, the github action flow (terraform-docs) added it